### ISSUE TITLE
Send last confirmation e-mail

### ISSUE THREAD

**Issue by dbrgn** [SwissPy/swisspycon#60]
Send e-mail to all attendees 5 days in advance with the following information:
- [ ] Ask people to cancel if they cannot attend
- [ ] Make sure everyone brings 10 CHF, ideally a 10 CHF bill
- [ ] Let people know that there will be a social event
- [ ] Let people know about key signing event (#24) and that they have to prepare in advance if they want to attend

Anything else?

**Comment by href** [comment]
Caution people to not show up too late as they'll have to get through a queue. We can handle 120 people, but only if they don't all show up at the last minute.

**Comment by chiesax** [comment]
Maybe remember them it is the first prototype version and that we therefore ask them to be patient and tolerant...

**Comment by chiesax** [comment]
By the way I can handle this...

@href any chance we can send a mass mail using for instance the Thunderbird add-on https://addons.mozilla.org/en-us/thunderbird/addon/mail-merge/? I remember you have mentioned we may not use anything else except the web application... should we make a test?

**Comment by href** [comment]
No problem, you can use any kind of program to send the mails. I just worry that multiple users moving messages around using IMAP would lead to problems. Our mail accounts are usually linked to a single person so I this use case is unexplored.

But as long as you're just sending messages you are fine. You can simply create a new address and add the SMTP info (mail.seantis.ch with your username and password, port 587). Just note that there's a 100 e-mail cap per hour, so I suggest you send the e-mails in two batches.

**Comment by dbrgn** [comment]
I don't think Thunderbird is needed, a regular e-mail with receivers in BCC should be fine :)

**Comment by href** [comment]
> I don't think Thunderbird is needed, a regular e-mail with receivers in BCC should be fine :)

Sure. You want to do this in badges as well though. BCC leads to multiple e-mails sent at the back which go towards the 100 mails per hour limit.

**Comment by dbrgn** [comment]
Ah :)

**Comment by chiesax** [comment]
Good point! I will try anyhow with Thunderbird because you do not need to add all recipients but just provide a CSV file with the email addresses (which we already have). If this works it may be useful in the future...

**Comment by chiesax** [comment]
A first draft...

Dear,

we are looking forward to meet you next Friday at the Swiss Python Summit 2016! 

In case you are not attending the meeting, we kindly ask you to cancel your registration by sending an email at info@pythonsummit.ch as soon as possible.

Please read the following important informations:
- registration begins at 8:30, please register as soon as possible, 
- please have a 10CHF bill ready for the registration,
- you are invited to participate to the Social Event after the conference (... to be better specified)
- (info about key signing party, is this on the website somewhere?)

Remember that all organizers and helpers of this meeting are volunteers. Be part of this spirit and collaborate to make this first event a pleasant experience for everybody.

**Comment by jakeret** [comment]
Some suggestions

> Dear <name>
> 
> The first Swiss Python Summit is taking place very soon. We are preparing full steam ahead for the conference.
> 
> Here are some important notes: 
> - registration begins at 8:30, please register as soon as possible,
> - please have a 10CHF bill ready for the registration,
> - you are invited to participate to the Social Event after the conference (... to be better specified)
> - (info about key signing party, is this on the website somewhere?)
> 
> Remember that all organizers and helpers of this meeting are volunteers. Be part of this spirit and collaborate to make this first event a pleasant experience for everybody.
> 
> The conference is fully booked out and there is a long list of people hoping for a spare ticket. In case you can’t attend the conference , we kindly ask you to cancel your registration by sending an email at info@pythonsummit.ch as soon as possible.
> 
> Best regards
> 
> SPS16 team

**Comment by href** [comment]
Another one:

> please register as soon as possible

Should be:

> please come and register early to avoid a long queue

Also if we are indeed opening at 08:00 already we might as well tell everyone, or not?

**Comment by chiesax** [comment]
@jakeret: I am not sure the sentence in which we ask the participants to cancel should be at the end of the email. I am worried people who are not coming may then not see this reminder. But I understand it is not nice to start with this...

**Comment by chiesax** [comment]
... but maybe I am just too paranoid :)

**Comment by dbrgn** [comment]
> I am worried people who are not coming may then not see this reminder.

Good point! We should probably add a "TL;DR" note in the beginning, saying that if they cannot attend they should let us know as soon as possible, so that we can let more people in.

> please have a 10CHF bill ready for the registration,

I'd just write "10 CHF" instead of "a 10 CHF bill". We do have change, so 20 or 50 wouldn't be too bad either.

**Comment by chiesax** [comment]
What about this (I am not sure I fully understand the deep TL;DR meaning though ...):

We are very proud to annouce you that the first Swiss Python Summit is fully booked out! Our waiting list is growing bigger: so in case you are not attending, please let us know by replying to this email as soon as possible...

**Comment by dbrgn** [comment]
TL;DR -> Too Long; Didn't Read

Yep, I like that, @chiesax!

**Comment by chiesax** [comment]
Meaning it should be short?

**Comment by dbrgn** [comment]
https://en.wikipedia.org/wiki/Wikipedia:Too_long;_didn%27t_read

> The tl;dr label is sometimes used constructively by an author to introduce a short summation of a longer piece.

**Comment by jakeret** [comment]
>  We are very proud to annouce you that the first Swiss Python Summit is fully booked out! Our waiting list is growing bigger: so in case you are not attending, please let us know by replying to this email as soon as possible...

Sounds good. Right after the "full steam ahead"-sentence

**Comment by chiesax** [comment]
@adamcik: any suggestions for the communication regarding the key signing party?

**Comment by adamcik** [comment]
Updated https://github.com/SwissPy/swisspycon/issues/24#issuecomment-173524159 with a suggest text. Copying here as well:

> There will be a small signing party during the social event. So if you want to get more signatures for your GPG key be sure to bring a suitable form of identification and the fingerprint of your key. For instance a printout from keysheet.net or just bring a laptop with the key.

**Comment by dbrgn** [comment]
Maye make that

> There will be a small **key** signing party

And regarding the fingerprint, you should probably make it a "no computers allowed" event. It causes less hassle and it's easier for everyone.

> and the fingerprint of your key on a piece of paper. Such a printout can be created easily on keysheet.net.

**Comment by adamcik** [comment]
> There will be a small key signing party during the social event. If you want to get more signatures for your GPG key be sure to bring a suitable form of identification and paper copie(s) of your key's fingerprint. For instance a printout from http://keysheet.net

I've left out the idea about printing the sheet on site to try and keep this short and to the point. For the info on the page we can probably add that bit as well?

**Comment by dbrgn** [comment]
Yep, agree!

> copie(s)

That should probably be "copy" or "copies", as a "copie" doesn't exist :)

**Comment by adamcik** [comment]
Would make it `copies` in that case.

**Comment by chiesax** [comment]
So to summarize: 

Dear {name}

The first Swiss Python Summit is taking place very soon. We are preparing full steam ahead for the conference and are very proud to announce you that the summit is fully booked out! 

Our waiting list is growing bigger: so in case you are not attending, please let us know by replying to this email as soon as possible...

Here are some important notes:
- please come and register early to avoid a long queue
- please have 10CHF cash ready for the registration,
- after the conference there will be an aperitif party,
- there will be a small key signing party during the aperitif. If you want to get more signatures for your GPG key be sure to bring a suitable form of identification and paper copies of your key's fingerprint. For instance a printout from http://keysheet.net

Remember that all organizers and helpers of this meeting are volunteers. Be part of this spirit and collaborate to make this first event a pleasant experience for everybody.

Best regards

SPS16 team

**Comment by chiesax** [comment]
I will send this out Friday or Saturday, your last comments are welcome...

**Comment by dbrgn** [comment]
- Instead of "Social Event" maybe we should write "Apéro"? Otherwise people will wonder what that includes.
- Maybe we should write that people should have the 10 CHF ready in cash?

**Comment by chiesax** [comment]
... updated

**Comment by adamcik** [comment]
My `copie(s)` typo is still in there btw :-)

**Comment by chiesax** [comment]
Sorry. Updated...

**Comment by dbrgn** [comment]
By the way, I updated the list of attendees, there's a new tab now where the attendees are grouped and cancelled reservations have been removed.

**Comment by dbrgn** [comment]
@chiesax I saw that you sent out a lot of confirmation e-mails. Did you take them from the current version of the attendees (including people from the waiting list that were added)?

**Comment by chiesax** [comment]
Yes from the Atendees tab. I skipped the speakers . Still have 40 mails to send.

**Comment by chiesax** [comment]
I will create a new tab with the csv I have used with Mail Merge.

**Comment by dbrgn** [comment]
Ok :) There are some that I converted from WAITINGLIST to REGULAR yesterday. Maybe you can include them too?

**Comment by chiesax** [comment]
Yeah I think they were included...
